### PR TITLE
Updating vSphere Cloud Provider Documentation

### PR DESCRIPTION
**Pull request by divyenpatel**
Summary of Change.

- Updated Introduction for vSphere Cloud Provider.
- Added link to vSphere Storage for Kubernetes Documentation.
- Added Slack Support Channel
- Removed resolved known issues, and added link to the list of known issues posted on vSphere Cloud Provider documentation site.
 

Preview - https://deploy-preview-5241--kubernetes-io-master-staging.netlify.com/docs/getting-started-guides/vsphere/

Please review: @BaluDontu @tusharnt @pdhamdhere

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/5241)

### CODE REVIEW

File: docs/getting-started-guides/vsphere.md [hunk 1]
```diff
@@ -12,24 +12,24 @@ This page covers how to get started with deploying Kubernetes on vSphere and det
...
+- Policy based management at granularity of container volumes
+- Volumes, Persistent Volumes, Storage Classes,  dynamic provisioning of volumes and scalable deployment of Stateful Apps with StatefulSets
 
-1. Volumes
-2. Persistent Volumes
-3. Storage Classes and provisioning of volumes.
-4. vSphere Storage Policy Based Management for Containers orchestrated by Kubernetes.
+For more detail visit [vSphere Storage for Kubernetes Documentation](https://vmware.github.io/vsphere-storage-for-kubernetes/documentation/index.html)
 
 Documentation for how to use vSphere managed storage can be found in the [persistent volumes user guide](/docs/concepts/storage/persistent-volumes/#vsphere) and the [volumes user guide](/docs/concepts/storage/volumes/#vspherevolume).
```
Comment (BaluDontu) [inline docs/getting-started-guides/vsphere.md]: The links in this paragraph are broken.

Comment (divyenpatel) [reply docs/getting-started-guides/vsphere.md]: checked with @BaluDontu, links are accessible from the preview site and not broken - https://deploy-preview-5241--kubernetes-io-master-staging.netlify.com/docs/getting-started-guides/vsphere/

**Comment by divyenpatel** [comment]
Addressed @BaluDontu review comment regarding Manifest files location.

**Comment by divyenpatel** [comment]
@steveperry-53 Can you help merge this PR?

**Comment by divyenpatel** [comment]
@chenopis can you help merging this PR?

**Pull request merged**